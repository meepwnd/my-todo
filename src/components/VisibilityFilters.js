import React from "react";
import FilterLink from "./FilterLink";
import { visibilityFilters } from "../actions/actionTypes";

const VisibilityFilters = () => (
  <div>
    <FilterLink filter={visibilityFilters.SHOW_ALL}>All</FilterLink>
    <FilterLink filter={visibilityFilters.SHOW_COMPLETED}>Completed</FilterLink>
    <FilterLink filter={visibilityFilters.SHOW_UNCOMPLETED}>
      Uncompleted
    </FilterLink>
  </div>
);

export default VisibilityFilters;
