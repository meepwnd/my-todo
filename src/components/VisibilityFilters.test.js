import React from "react";
import { shallow } from "enzyme";
import VisibilityFilters from "./VisibilityFilters";

describe("VisibilityFilters component", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<VisibilityFilters />);
    expect(wrapper).toMatchSnapshot();
  });
});
