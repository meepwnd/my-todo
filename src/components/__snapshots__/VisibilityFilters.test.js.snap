// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisibilityFilters component should render correctly 1`] = `
<div>
  <Connect(Link)
    filter="SHOW_ALL"
  >
    All
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_UNCOMPLETED"
  >
    Uncompleted
  </Connect(Link)>
</div>
`;
